src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY_EVENT_NAME = "buy";
+    private static final String SUPPLY_EVENT_NAME = "supply";
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMMOUNT = 1;
+    private static int supplyTotal = 0;
+    private static int buyTotal = 0;
```

Comment:
lets make supplyTotal and buyTotal parameters as local parameters (method readFile() could return array of 2 elements buy and supply)